INSTALL.md

```diff
@@ -212,6 +215,27 @@ back to the project source-code directory. That way, you can edit the source cod
 without reinstalling the package after a small change.
 * `--single-version-externally-managed` is an option for Python packages instructing the setup tools module to create a package the host's package manager can easily manage.
 
+## Build from Sources with `conda-build`
+
+Intel(R) Extension for Scikit-learn* is easily built from the sources using only one command and `conda-build` utility. 
+
+### Prerequisites for `conda-build`
+
+* any `conda` distribution (`miniforge` is recommended)
+* `conda-build` and `conda-verify` installed in a conda environment
+* (Windows only) Microsoft Visual Studio* 2022
+* (optional) Intel(R) oneAPI DPC++/C++ Compiler
+
+In order to enable DPC++ interfaces support, you need to set `DPCPPROOT` environment variable pointing to DPC++/C++ Compiler distribution.
```

Comment:
Please add commentary here about the divergence of windows and linux for dpc support. Also maybe MPI depending on the discussion in the other review comment.

Reply:
I added OS specific details for DPC compiler in INSTALL.md.